diff --git a/node_modules/react-native/Libraries/Components/ActivityIndicator/ActivityIndicator.js b/node_modules/react-native/Libraries/Components/ActivityIndicator/ActivityIndicator.js
index 2855cf4..efaeccb 100644
--- a/node_modules/react-native/Libraries/Components/ActivityIndicator/ActivityIndicator.js
+++ b/node_modules/react-native/Libraries/Components/ActivityIndicator/ActivityIndicator.js
@@ -153,10 +153,8 @@ const ActivityIndicator = (
 ```
 */
 
-const ActivityIndicatorWithRef: component(
-  ref: React.RefSetter<HostComponent<empty>>,
-  ...props: Props
-) = React.forwardRef(ActivityIndicator);
+const ActivityIndicatorWithRef = React.forwardRef(ActivityIndicator);
+
 ActivityIndicatorWithRef.displayName = 'ActivityIndicator';
 
 const styles = StyleSheet.create({
diff --git a/node_modules/react-native/Libraries/Components/Button.js b/node_modules/react-native/Libraries/Components/Button.js
index 1b5c0b6..8468953 100644
--- a/node_modules/react-native/Libraries/Components/Button.js
+++ b/node_modules/react-native/Libraries/Components/Button.js
@@ -285,10 +285,8 @@ const Touchable: typeof TouchableNativeFeedback | typeof TouchableOpacity =
 
 type ButtonRef = React.ElementRef<typeof Touchable>;
 
-const Button: component(
-  ref: React.RefSetter<ButtonRef>,
-  ...props: ButtonProps
-) = React.forwardRef((props: ButtonProps, ref: React.RefSetter<ButtonRef>) => {
+const Button = React.forwardRef((props, ref) => {
+
   const {
     accessibilityLabel,
     accessibilityState,
diff --git a/node_modules/react-native/Libraries/Components/LayoutConformance/LayoutConformance.js b/node_modules/react-native/Libraries/Components/LayoutConformance/LayoutConformance.js
index 9b3ebd4..78834be 100644
--- a/node_modules/react-native/Libraries/Components/LayoutConformance/LayoutConformance.js
+++ b/node_modules/react-native/Libraries/Components/LayoutConformance/LayoutConformance.js
@@ -48,9 +48,10 @@ function UnimplementedLayoutConformance(props: Props): React.Node {
   return props.children;
 }
 
-export default (isFabricUIManagerInstalled
+export default isFabricUIManagerInstalled
   ? LayoutConformance
-  : UnimplementedLayoutConformance) as component(...Props);
+  : UnimplementedLayoutConformance;
+
 
 const styles = StyleSheet.create({
   container: {
diff --git a/node_modules/react-native/Libraries/Components/Pressable/Pressable.js b/node_modules/react-native/Libraries/Components/Pressable/Pressable.js
index 68b2574..4cd21b9 100644
--- a/node_modules/react-native/Libraries/Components/Pressable/Pressable.js
+++ b/node_modules/react-native/Libraries/Components/Pressable/Pressable.js
@@ -363,7 +363,5 @@ function usePressState(forcePressed: boolean): [boolean, (boolean) => void] {
 const MemoedPressable = React.memo(React.forwardRef(Pressable));
 MemoedPressable.displayName = 'Pressable';
 
-export default (MemoedPressable: component(
-  ref: React.RefSetter<React.ElementRef<typeof View>>,
-  ...props: Props
-));
+export default MemoedPressable;
+
diff --git a/node_modules/react-native/Libraries/Components/ProgressBarAndroid/ProgressBarAndroid.android.js b/node_modules/react-native/Libraries/Components/ProgressBarAndroid/ProgressBarAndroid.android.js
index ca5c4fb..05ee707 100644
--- a/node_modules/react-native/Libraries/Components/ProgressBarAndroid/ProgressBarAndroid.android.js
+++ b/node_modules/react-native/Libraries/Components/ProgressBarAndroid/ProgressBarAndroid.android.js
@@ -78,12 +78,7 @@ export type ProgressBarAndroidProps = $ReadOnly<{|
  * },
  * ```
  */
-const ProgressBarAndroidWithForwardedRef: component(
-  ref: React.RefSetter<
-    React.ElementRef<typeof ProgressBarAndroidNativeComponent>,
-  >,
-  ...props: ProgressBarAndroidProps
-) = React.forwardRef(function ProgressBarAndroid(
+const ProgressBarAndroidWithForwardedRef = React.forwardRef(function ProgressBarAndroid(
   {
     // $FlowFixMe[incompatible-type]
     styleAttr = 'Normal',
